packages/gen_package_json.ts

```diff
@@ -0,0 +1,53 @@
+import * as fs from "fs";
+import * as yargs from "yargs";
+
```

Comment:
supernit: these double empty lines are a bit odd, is prettier not cleaning this file up?